### ISSUE TITLE
[FEATURE] Chronological feed

### ISSUE THREAD

**Issue by MrNabmas** [eni9889/IG-PP-Issues#179]
add an option for a chronological feed OR add an option to turn it on so any pictures after it was turned on will be in order

**Comment by D33dotexe** [comment]
Yes please god i thought i would get used to instagrams non-chronological style, but it’s been well over a year and i still absolutely hate it.

**Comment by apricity-spam** [comment]
I would really appreciate if this feature was added in the near future as the non-chronological feed is really annoying and should've never been forced upon the user.
